templates/kubernetes/overlays/staging/ingress.yml

```diff
@@ -13,7 +13,31 @@ metadata:
     external-dns.alpha.kubernetes.io/hostname: <% index .Params `stagingBackendSubdomain` %><% index .Params `stagingHostRoot` %>
     # CORS
     nginx.ingress.kubernetes.io/enable-cors: "true"
-    nginx.ingress.kubernetes.io/cors-allow-origin: "https://<% index .Params `stagingFrontendSubdomain` %><% index .Params `stagingHostRoot` %>/"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
```

Comment:
This should be split out into a dev ingress, we don't want this behaviour on a real staging environment.